yarn-project/end-to-end/src/e2e_block_building.test.ts

```diff
@@ -420,13 +418,27 @@ describe('e2e_block_building', () => {
     afterAll(() => teardown());
 
     it('calls a method with nested encrypted logs', async () => {
-      const thisWallet = new AccountWalletWithSecretKey(pxe, ownerWallet, owner.secret, owner.salt);
```

Comment:
While originally we manually decrypted here now I call the getPrivateEvents endpoint.